neuraxle/base.py

```diff
@@ -683,8 +561,29 @@ def hash(self, current_ids, hyperparameters, data_inputs: Any = None) -> List[st
         """
         for h in self.hashers:
             current_ids = h.hash(current_ids, hyperparameters, data_inputs)
+
         return current_ids
 
+    def summary_hash(self, current_ids, hyperparameters, data_inputs) -> str:
```

Comment:
It doesn't look like you've applied review comments if there is still this method in BaseStep.